client/src/components/workspace-nav/CollectionListItem.jsx

```diff
@@ -18,4 +24,5 @@ export default class CollectionListItem extends Component {
 
 CollectionListItem.propTypes = {
   collection: PropTypes.object,
+  pathname: PropTypes.string,
```

Comment:
This is more of a general comment which we can look out for in the future. Above we call  `this.props.pathname.indexOf(...)` without checking if `pathname` is `null/undefined`. This is fine, but should we in that case require that `pathname` is a string: `PropTypes.string.isRequired`.